code/modules/cargo/bounties/security.dm

```diff
@@ -11,3 +11,23 @@
 	reward = 2000
 	required_count = 3
 	wanted_types = list(/obj/machinery/recharger)
+
+/datum/bounty/item/security/seclocker
```

Comment:
Is this just the empty locker? I don't think these are really worth a reward.

Reply:
They lock and are armored, as well as sec only. They cant be replaced. I think cargo will *have* to make  for them